client/js/screens/BeginChallenge/styles.js

```diff
@@ -2,7 +2,61 @@ import { StyleSheet } from "react-native";
 import { colors, fonts } from "../../config/styles";
 
 const styles = StyleSheet.create({
- 
+  center: {
+    alignItems: "center",
+    flex: 1.25,
+    justifyContent: "space-around",
+    paddingTop: 120
+  },
+  flexText: {
+    alignItems: "center",
+    flex: 0.9,
+    justifyContent: "space-between",
+    paddingBottom: 60,
+    paddingTop: 30
```

Comment:
should not apply this much padding. please use flexbox properties

Reply:
Resolved